amazonia/classes/asg.py

```diff
@@ -106,16 +106,18 @@ def create_asg(self, title, minsize, maxsize, subnets, load_balancer, keypair, i
                 self.sns_notification_configurations = self.trop_asg.NotificationConfigurations = \
```

Comment:
Here is the other equals equals bit! I knew we'd find it! :)

` self.sns_notification_configurations = self.trop_asg.NotificationConfigurations = \`

can probably be simplified to `self.trop_asg.NotificationConfigurations = \` as long as `self.sns_notification_configurations` isn't referred to anywhere else?